mnefun/_report.py

```diff
@@ -0,0 +1,373 @@
+                    mlab.options.offscreen = True
+                    tempdir = mne.utils._TempDir()
+                    if len(mne.pick_types(sss_info)):
+                        coord_frame = 'meg'
+                    else:
+                        coord_frame = 'head'
+                    try:
+                        fig = mlab.figure(bgcolor=(0., 0., 0.),
+                                          size=(1000, 1000))
+                        kwargs = dict(
+                            info=sss_info, subjects_dir=subjects_dir, bem=bem,
+                            dig=True, coord_frame=coord_frame, show_axes=True,
+                            fig=fig, trans=trans, src=src)
+                        try_surfs = ['head-dense', 'head', 'inner_skull']
+                        for surf in try_surfs:
+                            try:
+                                mne.viz.plot_alignment(surfaces=surf, **kwargs)
+                            except Exception:
+                                pass
+                            else:
+                                break
+                        else:
+                            raise RuntimeError('Could not plot any surface '
+                                               'for alignment:\n%s'
+                                               % (try_surfs,))
+                        fig.scene.parallel_projection = True
+                        images = list()
+                        for ai, angle in enumerate([180, 90, 0]):
+                            mlab.view(angle, 90, focalpoint=(0., 0., 0.),
+                                      distance=0.6, figure=fig)
+                            view = mlab.screenshot(figure=fig)
+                            view = view[:, (view != 0).any(0).any(-1)]
+                            view = view[(view != 0).any(1).any(-1)]
+                            images.append(op.join(tempdir, '%s.png' % ai))
+                            imsave(images[-1], view)
+                        mlab.close(fig)
+                        report.add_images_to_section(
+                            images, captions=captions, section=section)
+                    finally:
+                        del tempdir
+                        mlab.options.offscreen = offscreen
+                print('%5.1f sec' % ((time.time() - t0),))
+            else:
+                print('    %s skipped' % section)
+            #
+            # BEM
+            #
+            section = 'BEM'
+            if p.report_params.get('bem', True):
+                caption = '%s: %s' % (section, struc)
+                bem, src, trans, _ = _get_bem_src_trans(
+                    p, raw.info, subj, struc)
+                if not bem['is_sphere']:
+                    t0 = time.time()
+                    print(('    %s ... ' % section).ljust(ljust), end='')
+                    report.add_bem_to_section(struc, caption, section,
+                                              decim=10, n_jobs=1)
```

Comment:
T1 should direct to `fsaverage` for cases without MRI

>   File "/home/ktavabi/Projects/mnefun/mnefun/_report.py", line 264, in gen_html_report
>     decim=10, n_jobs=1)
>   File "/home/ktavabi/Projects/mne-python/mne/report.py", line 1069, in add_bem_to_section
>     caption=caption)
>   File "/home/ktavabi/Projects/mne-python/mne/report.py", line 1758, in _render_bem
>     return self._render_image_png(mri_fname, cmap='gray')
>   File "/home/ktavabi/Projects/mne-python/mne/report.py", line 1543, in _render_image_png
>     nim = nib.load(image)
>   File "/home/ktavabi/miniconda3/envs/andrea-anders/lib/python2.7/site-packages/nibabel/loadsave.py", line 40, in load
>     raise FileNotFoundError("No such file: '%s'" % filename)
> nibabel.py3k.FileNotFoundError: No such file: '/media/ktavabi/ALAYA/data/freesurfer/sub-nbwr301/mri/T1.mgz'

Reply:
We have no guarantee of which subject is used as a surrogate, so I think for now I'll skip it with a warning and users can do `bem=False`

Reply:
Pushed a commit, feel free to try again